core/src/types/eth_bridge_pool.rs

```diff
@@ -173,16 +218,26 @@ pub struct PendingTransfer {
     pub gas_fee: GasFee,
 }
 
+/// Construct a token address from an ERC20 address.
+pub fn erc20_token_address(address: &EthAddress) -> Address {
```

Comment:
This isn't really just a bridge pool thing, it's a bit more general than that. Maybe it could go live with tokens instead

Reply:
there's unfortunately an issue with this - the `PendingTransfer::token_address` is using it, which itself is used from other crates